src/components/BookForm.js

```diff
@@ -24,9 +26,12 @@ const BookForm = () => {
 
   return (
     <form className="book-form" onSubmit={handleSubmit}>
-      <input name="title" placeholder="title" onChange={handleInputChange} />
-      <input name="author" placeholder="author" onChange={handleInputChange} />
-      <button type="submit">Submit</button>
+      <div className="book-form__title">ADD NEW BOOK</div>
+      <div className="flex flex-col md:grid grid-cols-4 gap-x-9 gap-y-4">
+        <input className="book-form__input col-span-2" name="title" placeholder="title" onChange={handleInputChange} />
+        <input className="book-form__input" name="author" placeholder="author" onChange={handleInputChange} />
+        <Button buttonType="submit">Submit</Button>
```

Comment:
- [ ] Please reduce the width of this button and give it the name `ADD BOOK` to match the Zeplin design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/178959218-93f8f6d1-110b-47ae-91bf-d89308ba94df.png)

### Zeplin design

![image](https://user-images.githubusercontent.com/83803414/178959269-7c6bc200-618d-4ea5-9e42-0d5cb6fdd103.png)